index.js

```diff
@@ -1,6 +1,7 @@
 const Bottleneck = require('bottleneck')
 
 const limiter = new Bottleneck({ maxConcurrent: 1, minTime: 0 })
+const ignoredAccounts = (process.env.IGNORED_ACCOUNTS || '').split(',')
```

Comment:
the chances are probably tiny, but to be sure maybe add a `.toLowerCase()` before the split?